packages/extensions-framework/src/Container.ts

```diff
@@ -0,0 +1,62 @@
+import { Container as InversifyContainer, interfaces } from "inversify";
+import { EventEmitter } from "events";
+
+const BOUND = "BOUND";
```

Comment:
Is there a reason why we don't use symbols here?

Reply:
Yes, `EventEmitter` types don't support Symbols. We could contribute, though it was out of scope

Reply:
Sure, I guess it is :)